pkg/hooks/launch.go

```diff
@@ -657,7 +658,7 @@ func getInodeNumber(pid int) uint64 {
 
 	f, err := os.Stat(filepath)
 	if err != nil {
-		fmt.Errorf("%v failed to get the inode number or namespace Id:", Emoji, err)
+		log.Errorf("%v failed to get the inode number or namespace Id: %w", Emoji, err)
 		return 0
 	}
 	// Dev := (f.Sys().(*syscall.Stat_t)).Dev
```

Comment:
instead of using log function, pass the logger from ``launch.go```

Reply:
thanks! I fixed it.
[use logger](https://github.com/keploy/keploy/pull/1351/commits/96041c591a798a9c0e0485edf693ef6981996ef7)